tests/__mocks__/organizationHandlers.js

```diff
@@ -155,6 +155,19 @@ export const organizationHandlers = [
     }
     return res(ctx.status(200));
   }),
+  rest.get(`${iotManagerBaseURL}/events`, (req, res, ctx) => {
```

Comment:
Nit: `req` is never used

Reply:
yep, but I need to get to the 2nd and 3rd argument and there's no named arguments in JS... just object destructuring (if an object would be passed in here)